dojo/importers/default_importer.py

```diff
@@ -226,7 +226,11 @@ def process_findings(
             # Process any endpoints on the endpoint, or added on the form
             self.process_endpoints(finding, self.endpoints_to_add)
             # Parsers must use unsaved_tags to store tags, so we can clean them
-            finding.tags = clean_tags(finding.unsaved_tags)
+            cleaned_tags = clean_tags(finding.unsaved_tags)
+            if isinstance(cleaned_tags, list):
+                finding.tags.set(cleaned_tags)
+            elif isinstance(cleaned_tags, str):
+                finding.tags.set([cleaned_tags])
```

Comment:
reimport has similar code that needs to be changed:
https://github.com/valentijnscholten/django-DefectDojo/blob/1ba1122954b6fa214862e23345373f75434cdaa1/dojo/importers/default_reimporter.py#L691-L693